ddev/ddev
# fix: respect `~/bin` for composer commands, for #6627

## The Issue

This comment:

- https://github.com/ddev/ddev/pull/6688#issuecomment-2457253832

For:

- #6627

## How This PR Solves The Issue

It turned out that when you pass `PATH=~/bin:...` as env, it doesn't understand what `~` means.

## Manual Testing Instructions

```
ddev composer require composer/composer:v2.8.0 --dev
```

Add this to `.ddev/config.yaml`:

```
hooks:
    post-start:
        - exec: mkdir -p ~/bin && ln -sf /usr/local/bin/composer ~/bin/composer
```

```
ddev restart
ddev composer -V # should be the same as in /usr/local/bin/composer
```

## Automated Testing Overview



## Release/Deployment Notes
